### PR TITLE
added find all action

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
issue #100

### CODE REVIEW

File: src/actions/Find.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { ActionReturnValue, Graph, TypedSelection } from "../Types";
+import { commands } from "vscode";
+import GetText from "./GetText";
+
+class FindBase extends GetText {
```
Comment (pokey) [inline src/actions/Find.ts:5]: Why does this extend `GetText`? If you need to get text, I'd just do it directly, or if you'd like to use the action just call it from the graph. See eg `carve`

File: src/actions/GetText.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import {
+  Action,
+  ActionPreferences,
+  ActionReturnValue,
+  Graph,
+  TypedSelection,
+} from "../Types";
+import displayPendingEditDecorations from "../editDisplayUtils";
+
+export default class GetText implements Action {
```
Comment (pokey) [inline src/actions/GetText.ts:10]: This one will be great to have, eg for "phones"

**Review by pokey — COMMENTED** [review]
One minor tweak but otherwise looks good!

File: src/actions/Find.ts (hunk 1 above)
Comment (AndreasArvidsson) [reply src/actions/Find.ts:5]: I was thinking that we could have optional constructor arguments. So a subclass is better in that regard then calling via graph, but for now it doesn't really matter so calling via graph is fine if you prefer that.

File: src/actions/GetText.ts (hunk 2 above)
Comment (AndreasArvidsson) [reply src/actions/GetText.ts:10]: Yes phones and formatters. I find myself missing both.

File: src/actions/Find.ts (hunk 1 above)
Comment (pokey) [reply src/actions/Find.ts:5]: Sorry I'm not sure I follow. You're not passing any constructor arguments to the base class, and it doesn't have any optional arguments

Comment (AndreasArvidsson) [reply src/actions/Find.ts:5]: Not yet. For phones and such. Was a future provision. But for find it does t matter.

**Review by pokey — APPROVED** [review]

**Pull request merged**